integration/.golangci.yml

```diff
@@ -59,7 +65,7 @@ linters-settings:
         alias: dto
   # ineffassign
   lll:
-    line-length: 130
+    line-length: 175 # TODO reduce it to 130
```

Comment:
Is there any background behind that? Do we have any place in code which requires as to adjust it to 175?

Reply:
Yep, we have long lines in our integration tests. Before that PR, they were not checked due to mistake in the configuration